### PR TITLE
VZV | Years of Life Lost Bug

### PR DESCRIPTION
**Pull request by mddilley**
Closes part of https://github.com/cityofaustin/atd-data-tech/issues/2794 and https://github.com/cityofaustin/atd-data-tech/issues/2790

This PR updates the VZ ETL process to truncate the datasets before upserting all records on each run. It also improves the logging in the script. Last, I added the method that concats `crash_date` and `crash_time` fields in the demographics data which was already implemented for the crash record formatting.

There are also some formatting updates from Black.

#### Updated logging
Screen Shot 2020-06-24 at 10 26 08 AM

### CODE REVIEW

File: atd-etl/app/process/helpers_socrata.py [hunk 1]
```diff
@@ -288,24 +291,23 @@ def format_person_data(data, formatter_config):
...
     # Join records and format
     people_records = person_records + primary_person_records
     formatted_records = rename_record_columns(
-        people_records, formatter_config["columns_to_rename"])
-    formatted_records = flatten_hasura_response(
-        formatted_records)
+        people_records, formatter_config["columns_to_rename"]
+    )
+    formatted_records = flatten_hasura_response(formatted_records)
+    formatted_records = concatTimeAndDate(formatted_records)
```
Comment (mddilley) [inline atd-etl/app/process/helpers_socrata.py:311]: Concat date and time for `crash_date` field here

File: atd-etl/app/process_socrata_export.py [hunk 2]
```diff
@@ -66,6 +69,8 @@
 # For each config, get records from Hasura and upsert to Socrata until res is []
 for config in query_configs:
     print(f'Starting {config["table"]} table...')
+    print(f'Truncating {config["table"]} table...')
+    client.replace(config["dataset_uid"], [])
```
Comment (mddilley) [inline atd-etl/app/process_socrata_export.py:73]: Truncating datasets here

File: atd-etl/app/process/socrata_queries.py [hunk 3]
```diff
@@ -70,6 +70,8 @@
             unit_nbr
             crash {
                 crash_date
+                crash_time
+                crash_id
```
Comment (mddilley) [inline atd-etl/app/process/socrata_queries.py:74]: Added `crash_id` and `crash_time` to demographics datasets

Comment (mateoclarke) [reply atd-etl/app/process/socrata_queries.py:74]: 👍

**Pull request merged**